features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -116,9 +124,53 @@ fun RoomMemberListView(
     }
 }
 
+@Composable
+private fun RoomMemberList(
+    roomMembers: Async.Success<RoomMembers>,
+    onUserSelected: (RoomMember) -> Unit,
+) {
+    LazyColumn(modifier = Modifier.fillMaxWidth(), state = rememberLazyListState()) {
```

Comment:
From what I understood, a Composable should always take a modifier as a parameter with a default value, and the root component in the composable should use this modifier. Is it not the case?

Reply:
AFAIK what you mention is good practice whenever creating public, reusable composables (that create a public API, like the ones we have in the `designsystem` module).
For private composable functions like this there is no need of enforcing this pattern unless the modifier is actually needed up the hierarchy. @jmartinesp what's your take on this?

Reply:
I agree, if it's a private component we can always modify it and handle any side-effects ourselves. It's not very likely that the component will be reused and if it is, we can always include the modifier later if needed.